package.json

```diff
@@ -51,8 +51,7 @@
     }
   },
   "files": [
-    "dist",
-    "*.d.ts"
+    "dist"
   ],
   "typesVersions": {
```

Comment:
You already config path alias here. So it will pointing to `./dist/dist/index.d.ts`